src/s1/t3/n1/exercici1/Main.java

```diff
@@ -0,0 +1,29 @@
+package s1.t3.n1.exercici1;
+
+
+
+public class Main {
+    void main(String[] args) {
+        Month august = new Month(Month.MONTHS.august);
+        // Init year empty and fill manually according to instruction.
```

Comment:
You should remove the comments. 
Too many comments mean that your code is not so readable... that was what both teachers have said.

Reply:
Commenting an anti-pattern.